HeliPort/Appearance/StatusMenu.swift

```diff
@@ -312,9 +312,7 @@ final class StatusMenu: NSMenu, NSMenuDelegate {
             get_80211_state(&status)
 
             DispatchQueue.main.async {
-                if get_power_ret {
-                    self.isNetworkEnabled = powerState
-                }
+                self.isNetworkEnabled = get_power_ret
```

Comment:
`get_power_ret` only represents the result of the API call (succeeded or failed), the obtained power state (on or off) is in `powerState` var. The switch will be updated when the power state is successfully obtained here.